frontend/src/concepts/analyticsTracking/segmentIOUtils.tsx

```diff
@@ -108,6 +110,10 @@ export const fireIdentifyEvent = (properties: IdentifyEventProperties): void =>
       isAdmin: properties.isAdmin,
       canCreateProjects: properties.canCreateProjects,
     };
-    window.analytics.identify(properties.anonymousID, traits);
+    if (properties.userID) {
+      window.analytics.identify(properties.userID, traits);
+    } else {
+      window.analytics.identify(properties.anonymousID, traits);
+    }
```

Comment:
`anonymousID` is longer set:
```suggestion
    window.analytics.identify(properties.userID, traits);
```